.travis/build_examples.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+repo_path="$GOPATH/src/github.com/unidoc/unipdf-examples"
```

Comment:
Is this GOPATH based?  Would make sense to make this entirely module based?

Reply:
It is not. I can also clone the repository inside the unipdf repository in the build environment. The GO111MODULE is globally enabled.